.github/workflows/continuous-integration.yml

```diff
@@ -109,4 +109,5 @@ jobs:
         # the generated protocol test projects will use whatever the SDK thinks the version of smithy-kotlin should be
         sed -i "s/smithy-kotlin-version = .*$/smithy-kotlin-version = \"$SMITHY_KOTLIN_VERSION\"/" gradle/libs.versions.toml
         ./gradlew --parallel publishToMavenLocal
+        ./gradlew test jvmTest
```

Comment:
**Question**: Doesn't this run the Linux tests too, which we don't want?

Reply:
No, `allTests` would run the Linux suite, `test` alone does not

Reply:
Linux tests are disabled by our plugin right now: https://github.com/awslabs/aws-kotlin-repo-tools/blob/main/build-plugins/src/main/kotlin/aws/sdk/kotlin/gradle/kmp/ConfigureTargets.kt#L167

But also no `test` covers codegen and integration tests that use the `kotlin(jvm)` plugin, `jvmTest` covers KMP.